project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.8.2
```

Comment:
**question:** how tf did we end up removing this

Reply:
it was in a PR of @yisraelU's that I merged recently, I overlooked this change for some reason :/  see https://github.com/disneystreaming/smithy4s/pull/921